CMakeLists.txt

```diff
@@ -11,6 +13,9 @@ find_package(catkin REQUIRED COMPONENTS roscpp nodelet tf gazebo_plugins gazebo_
 find_package(Boost 1.40 COMPONENTS program_options date_time filesystem iostreams regex system thread REQUIRED)
 find_package(Protobuf REQUIRED)
 
+set(FPHSA_NAME_MISMATCHED TRUE)
```

Comment:
This just silences a warning which is there regardless of whether you use `gazebo`, `Gazebo`, or `GAZEBO`. This is a bug in the Gazebo find module; I've ticketed it here:

https://github.com/osrf/gazebo/issues/3081